controllers/cache/provision.go

```diff
@@ -18,10 +19,10 @@ import (
 // - perform any RedisFailovers downscale (left for downscale phase)
 // - perform any RedisFailovers upscale (left for upscale phase)
 // - perform any pod upgrade (left for rolling upgrade phase)
-func (redis *RedisReconciler) Deploy() error {
+func (redis *RedisReconciler) Deploy() (*lcm.CRStatus, error) {
 
 	if redis.HarborCluster.Spec.Redis.Kind == "external" {
-		return nil
+		return cacheUnknownStatus(),nil
```

Comment:
why return `UnknownStatus` when using `external mode`?

Reply:
Reconcile will flow to the `Readiness` controller, `Readiness` will use ping to  check external Redis.